tests/unit/models/builder-test.js

```diff
@@ -36,27 +38,36 @@ describe('models/builder.js', function() {
   }
 
   before(function() {
-    td.replace('../../../lib/utilities/will-interrupt-process', {
-      addHandler: td.function(),
-      removeHandler: td.function(),
-    });
-
     Builder = require('../../../lib/models/builder');
   });
 
+  beforeEach(function() {
+    willInterruptProcess.capture(new MockProcess());
+  }),
+
   afterEach(function() {
+    willInterruptProcess.release();
     if (builder) {
       return builder.cleanup();
     }
   });
 
   describe('copyToOutputPath', function() {
+    let command;
+
+    let parentPath = `..${path.sep}..${path.sep}`;
+
+    before(function() {
+      command = new BuildCommand(commandOptions());
+    });
+
     beforeEach(function() {
       return mkTmpDirIn(tmproot).then(function(dir) {
         tmpdir = dir;
         builder = new Builder({
           setupBroccoliBuilder,
           project: new MockProject(),
+          onProcessInterrupt: willInterruptProcess,
```

Comment:
I've seen @stefanpenner used `shutdown` term few times. Is `onShutdown` better term?